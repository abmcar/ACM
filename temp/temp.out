哥布林的头盔:
  display: '&f哥布林的头盔'
  lore:
    - '&8&m----------------------------'
    - '&8防具                   &f普通'
    - ''
    - '&7装备等级: &f1'
    - '&7生命力: &f120'
    - '&7生命恢复: &f3'
    - '&7装备类型: &f头盔'
    - ''
    - '&8&m----------------------------'
    - ''
    - '&7可打造'
    - '&7数值变化 生命力 + 8'
    - '&7所需材料 矛盾的结晶体 * 1'
    - '&7成功率 100'
    - ''
    - '耐久度: 不可损坏'
哥布林的衣服:
  display: '&f哥布林的衣服'
  lore:
    - '&8&m----------------------------'
    - '&8防具                   &f普通'
    - ''
    - '&7装备等级: &f1'
    - '&7生命力: &f120'
    - '&7生命恢复: &f3'
    - '&7装备类型: &f衣服'
    - ''
    - '&8&m----------------------------'
    - ''
    - '&7可打造'
    - '&7数值变化 生命力 + 8'
    - '&7所需材料 矛盾的结晶体 * 1'
    - '&7成功率 100'
    - ''
    - '耐久度: 不可损坏'
哥布林的裤子:
  display: '&f哥布林的裤子'
  lore:
    - '&8&m----------------------------'
    - '&8防具                   &f普通'
    - ''
    - '&7装备等级: &f1'
    - '&7生命力: &f120'
    - '&7生命恢复: &f3'
    - '&7装备类型: &f裤子'
    - ''
    - '&8&m----------------------------'
    - ''
    - '&7可打造'
    - '&7数值变化 生命力 + 8'
    - '&7所需材料 矛盾的结晶体 * 1'
    - '&7成功率 100'
    - ''
    - '耐久度: 不可损坏'
哥布林的鞋子:
  display: '&f哥布林的鞋子'
  lore:
    - '&8&m----------------------------'
    - '&8防具                   &f普通'
    - ''
    - '&7装备等级: &f1'
    - '&7生命力: &f120'
    - '&7生命恢复: &f3'
    - '&7装备类型: &f鞋子'
    - ''
    - '&8&m----------------------------'
    - ''
    - '&7可打造'
    - '&7数值变化 生命力 + 8'
    - '&7所需材料 矛盾的结晶体 * 1'
    - '&7成功率 100'
    - ''
    - '耐久度: 不可损坏'
