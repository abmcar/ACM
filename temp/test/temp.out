  1:
  - "物理伤害: +{8|8} @100"
  - "生命力: +{120|120} @100"
  2:
  - "物理伤害: +{1|1} @100"
  - "生命力: +{16|16} @100"
  3:
  - "物理伤害: +{1|1} @100"
  - "生命力: +{16|16} @100"
  4:
  - "物理伤害: +{1|1} @100"
  - "生命力: +{16|16} @100"
  5:
  - "物理伤害: +{1|1} @100"
  - "生命力: +{16|16} @100"
  6:
  - "物理伤害: +{1|1} @100"
  - "生命力: +{16|16} @100"
  7:
  - "物理伤害: +{1|1} @100"
  - "生命力: +{16|16} @100"
  8:
  - "物理伤害: +{1|1} @100"
  - "生命力: +{16|16} @100"
  9:
  - "物理伤害: +{1|1} @100"
  - "生命力: +{16|16} @100"
  10:
  - "物理伤害: +{1|1} @100"
  - "生命力: +{287|287} @100"
  11:
  - "物理伤害: +{2|2} @100"
  - "生命力: +{29|29} @100"
  12:
  - "物理伤害: +{2|2} @100"
  - "生命力: +{29|29} @100"
  13:
  - "物理伤害: +{2|2} @100"
  - "生命力: +{29|29} @100"
  14:
  - "物理伤害: +{2|2} @100"
  - "生命力: +{29|29} @100"
  15:
  - "物理伤害: +{2|2} @100"
  - "生命力: +{29|29} @100"
  16:
  - "物理伤害: +{2|2} @100"
  - "生命力: +{29|29} @100"
  17:
  - "物理伤害: +{2|2} @100"
  - "生命力: +{29|29} @100"
  18:
  - "物理伤害: +{2|2} @100"
  - "生命力: +{29|29} @100"
  19:
  - "物理伤害: +{2|2} @100"
  - "生命力: +{29|29} @100"
  20:
  - "物理伤害: +{2|2} @100"
  - "生命力: +{586|586} @100"
  21:
  - "物理伤害: +{3|3} @100"
  - "生命力: +{41|41} @100"
  22:
  - "物理伤害: +{3|3} @100"
  - "生命力: +{41|41} @100"
  23:
  - "物理伤害: +{3|3} @100"
  - "生命力: +{41|41} @100"
  24:
  - "物理伤害: +{3|3} @100"
  - "生命力: +{41|41} @100"
  25:
  - "物理伤害: +{3|3} @100"
  - "生命力: +{41|41} @100"
  26:
  - "物理伤害: +{3|3} @100"
  - "生命力: +{41|41} @100"
  27:
  - "物理伤害: +{3|3} @100"
  - "生命力: +{41|41} @100"
  28:
  - "物理伤害: +{3|3} @100"
  - "生命力: +{41|41} @100"
  29:
  - "物理伤害: +{3|3} @100"
  - "生命力: +{41|41} @100"
  30:
  - "物理伤害: +{3|3} @100"
  - "生命力: +{762|762} @100"
  31:
  - "物理伤害: +{4|4} @100"
  - "生命力: +{50|50} @100"
  32:
  - "物理伤害: +{4|4} @100"
  - "生命力: +{50|50} @100"
  33:
  - "物理伤害: +{4|4} @100"
  - "生命力: +{50|50} @100"
  34:
  - "物理伤害: +{4|4} @100"
  - "生命力: +{50|50} @100"
  35:
  - "物理伤害: +{4|4} @100"
  - "生命力: +{50|50} @100"
  36:
  - "物理伤害: +{4|4} @100"
  - "生命力: +{50|50} @100"
  37:
  - "物理伤害: +{4|4} @100"
  - "生命力: +{50|50} @100"
  38:
  - "物理伤害: +{4|4} @100"
  - "生命力: +{50|50} @100"
  39:
  - "物理伤害: +{4|4} @100"
  - "生命力: +{50|50} @100"
  40:
  - "物理伤害: +{4|4} @100"
  - "生命力: +{852|852} @100"
  41:
  - "物理伤害: +{5|5} @100"
  - "生命力: +{58|58} @100"
  42:
  - "物理伤害: +{5|5} @100"
  - "生命力: +{58|58} @100"
  43:
  - "物理伤害: +{5|5} @100"
  - "生命力: +{58|58} @100"
  44:
  - "物理伤害: +{5|5} @100"
  - "生命力: +{58|58} @100"
  45:
  - "物理伤害: +{5|5} @100"
  - "生命力: +{58|58} @100"
  46:
  - "物理伤害: +{5|5} @100"
  - "生命力: +{58|58} @100"
  47:
  - "物理伤害: +{5|5} @100"
  - "生命力: +{58|58} @100"
  48:
  - "物理伤害: +{5|5} @100"
  - "生命力: +{58|58} @100"
  49:
  - "物理伤害: +{5|5} @100"
  - "生命力: +{58|58} @100"
  50:
  - "物理伤害: +{5|5} @100"
  - "生命力: +{856|856} @100"
  51:
  - "物理伤害: +{7|7} @100"
  - "生命力: +{74|74} @100"
  52:
  - "物理伤害: +{7|7} @100"
  - "生命力: +{74|74} @100"
  53:
  - "物理伤害: +{7|7} @100"
  - "生命力: +{74|74} @100"
  54:
  - "物理伤害: +{7|7} @100"
  - "生命力: +{74|74} @100"
  55:
  - "物理伤害: +{7|7} @100"
  - "生命力: +{74|74} @100"
  56:
  - "物理伤害: +{7|7} @100"
  - "生命力: +{74|74} @100"
  57:
  - "物理伤害: +{7|7} @100"
  - "生命力: +{74|74} @100"
  58:
  - "物理伤害: +{7|7} @100"
  - "生命力: +{74|74} @100"
  59:
  - "物理伤害: +{7|7} @100"
  - "生命力: +{74|74} @100"
  60:
  - "物理伤害: +{7|7} @100"
  - "生命力: +{994|994} @100"
  61:
  - "物理伤害: +{9|9} @100"
  - "生命力: +{85|85} @100"
  62:
  - "物理伤害: +{9|9} @100"
  - "生命力: +{85|85} @100"
  63:
  - "物理伤害: +{9|9} @100"
  - "生命力: +{85|85} @100"
  64:
  - "物理伤害: +{9|9} @100"
  - "生命力: +{85|85} @100"
  65:
  - "物理伤害: +{9|9} @100"
  - "生命力: +{85|85} @100"
  66:
  - "物理伤害: +{9|9} @100"
  - "生命力: +{85|85} @100"
  67:
  - "物理伤害: +{9|9} @100"
  - "生命力: +{85|85} @100"
  68:
  - "物理伤害: +{9|9} @100"
  - "生命力: +{85|85} @100"
  69:
  - "物理伤害: +{9|9} @100"
  - "生命力: +{85|85} @100"
  70:
  - "物理伤害: +{9|9} @100"
  - "生命力: +{960|960} @100"
  71:
  - "物理伤害: +{11|11} @100"
  - "生命力: +{92|92} @100"
  72:
  - "物理伤害: +{11|11} @100"
  - "生命力: +{92|92} @100"
  73:
  - "物理伤害: +{11|11} @100"
  - "生命力: +{92|92} @100"
  74:
  - "物理伤害: +{11|11} @100"
  - "生命力: +{92|92} @100"
  75:
  - "物理伤害: +{11|11} @100"
  - "生命力: +{92|92} @100"
  76:
  - "物理伤害: +{11|11} @100"
  - "生命力: +{92|92} @100"
  77:
  - "物理伤害: +{11|11} @100"
  - "生命力: +{92|92} @100"
  78:
  - "物理伤害: +{11|11} @100"
  - "生命力: +{92|92} @100"
  79:
  - "物理伤害: +{11|11} @100"
  - "生命力: +{92|92} @100"
  80:
  - "物理伤害: +{11|11} @100"
  - "生命力: +{754|754} @100"
  81:
  - "物理伤害: +{13|13} @100"
  - "生命力: +{96|96} @100"
  82:
  - "物理伤害: +{13|13} @100"
  - "生命力: +{96|96} @100"
  83:
  - "物理伤害: +{13|13} @100"
  - "生命力: +{96|96} @100"
  84:
  - "物理伤害: +{13|13} @100"
  - "生命力: +{96|96} @100"
  85:
  - "物理伤害: +{13|13} @100"
  - "生命力: +{96|96} @100"
  86:
  - "物理伤害: +{13|13} @100"
  - "生命力: +{96|96} @100"
  87:
  - "物理伤害: +{13|13} @100"
  - "生命力: +{96|96} @100"
  88:
  - "物理伤害: +{13|13} @100"
  - "生命力: +{96|96} @100"
  89:
  - "物理伤害: +{13|13} @100"
  - "生命力: +{96|96} @100"
  90:
  - "物理伤害: +{13|13} @100"
  - "生命力: +{376|376} @100"
  91:
  - "物理伤害: +{15|15} @100"
  - "生命力: +{95|95} @100"
  92:
  - "物理伤害: +{15|15} @100"
  - "生命力: +{95|95} @100"
  93:
  - "物理伤害: +{15|15} @100"
  - "生命力: +{95|95} @100"
  94:
  - "物理伤害: +{15|15} @100"
  - "生命力: +{95|95} @100"
  95:
  - "物理伤害: +{15|15} @100"
  - "生命力: +{1197|1197} @100"
  96:
  - "物理伤害: +{18|18} @100"
  - "生命力: +{113|113} @100"
  97:
  - "物理伤害: +{18|18} @100"
  - "生命力: +{113|113} @100"
  98:
  - "物理伤害: +{18|18} @100"
  - "生命力: +{113|113} @100"
  99:
  - "物理伤害: +{18|18} @100"
  - "生命力: +{113|113} @100"
  100:
  - "物理伤害: +{18|18} @100"
  - "生命力: +{113|113} @100"
